src/configs/config.aws.js

```diff
@@ -5,7 +5,9 @@ Planetiler tile server, hosted at AWS
 */
 const OPENMAPTILES_URL =
   "https://6ug7hetxl9.execute-api.us-east-2.amazonaws.com/data/v3.json";
+const SHIELD_TEXT_HALO_COLOR = null;
```

Comment:
I'd suggest we briefly document the purpose and usage of the variable with a comment.  Prefilling with a useful value and then commenting it out also might be nice.  Then you just have to un-comment to turn this debug feature on.

```suggestion

// Uncomment this variable to override the shield text halo color.  Useful while testing shield design changes.
// Accepts an HTML color name, hex code, or other CSS color value.
//const SHIELD_TEXT_HALO_COLOR = "magenta";
```

I'd also suggest the variable be named a bit more descriptively so it's clear this is a debug/override/test feature, not something that gets used in production.  `SHIELD_TEXT_HALO_COLOR_DEBUG_OVERRIDE` is rather long long.  Maybe just `SHIELD_TEXT_HALO_COLOR_OVERRIDE`?